modules/installation-supported-aws-machine-types.adoc

```diff
@@ -407,8 +412,34 @@ The following Amazon Web Services (AWS) instance types are supported with {produ
 
 |===
 ====
+endif::[]
+
+ifdef::localzone[]
+Each Amazon Web Services (AWS) Local Zone supports specific instance types for an AWS service. 
+
+[NOTE]
+====
+Red Hat did not fully test the instance types included in the following chart for your AWS instance on {product-title}. 
+====
+
+.Machine types based on x86_64 architecture for AWS Local Zones
+[%collapsible]
+====
+* m6i.*
+* m5.*
+* r5.*
+* c5.*
+* c5d.*
+* t3.*
+====
+
+[NOTE]
+====
+The `openshift-installer` scans the list of instance type in the order defined in the previous table, with the exception of the `t3` instance type. If you want to use the `t3` instance type, you must manually specify it in the `install-config.yaml` file. 
```

Comment:
@mtulio is this the scan list? https://github.com/openshift/installer/pull/6371/files#diff-7abecca62150eb15cb02e930cd045f4d6782f3f2b3a933d3cb67c89bc6369026R64-R70

Reply:
@mtulio @dfitzmau  Sorry I missed the previous comment [1], my suggestion is if we add this node, then we should keep the order exactly the same as the installer code [2], otherwise user may get an unexpected configuration. 

I prefer to remove this note, since we did not mention this logic in other AWS install chapters. WDYT?

[1] https://github.com/openshift/openshift-docs/pull/57057#discussion_r1134051743
[2] https://github.com/openshift/installer/pull/6371/files#diff-7abecca62150eb15cb02e930cd045f4d6782f3f2b3a933d3cb67c89bc6369026R64-R70

Reply:
> I prefer to remove this note, since we did not mention this logic in other AWS install chapters. WDYT?

@yunjiang29 +1 for this statement. I would prefer to say something like "the installer will chose the preferred instance based on the tested ones", which is what the algorithm does.

But I am also ok to remove that note and avoid unexpected behaviors

Reply:
Hi @yunjiang29 and @mtulio , link [2] lists `m5` twice and does not list `t3`. Should I remove "t3" from the list and place "m5" farther down the line? The note centers around the `t3` instance type because it's a type a user must manually select. 

I'd prefer to delete the note, because I would need to go more granular if I retain it. In other words, the installer chooses instance types in the following order: (1) tested, (2) supported, and then? The `t3` type should also be called out as a manual selection, because the installer cannot select it. 

Please correct me if I'm wrong on any of the above points.

Reply:
It looks like we are on the same page - remove the note :)




> Hi @yunjiang29 and @mtulio , link [2] lists `m5` twice and does not list `t3`. Should I remove "t3" from the list and place "m5" farther down the line? The note centers around the `t3` instance type because it's a type a user must manually select.

@dfitzmau  `t3` is not on the list is b/c it's not recommended for production usage (@mtulio correct me if I'm wrong), but since this list shows _tested instance type family_, so I think keeping it on the list is OK, user can choose anyone they want.   Order by alphabet is a good choice, it's simple and easy to maintain.

Reply:
Hi @yunjiang29 . I removed the notes, but added a new note to explain the use of the `t3` instance type. 

I reordered the list in alphabetic order. 

Please let me know your thoughts.

Reply:
> (@mtulio correct me if I'm wrong)

You're correct @yunjiang29 . 

LGTM the fixed statement.